usage_examples/tests/test_readme_examples.py

```diff
@@ -87,3 +91,45 @@ def test_example_3_python():
     )
     doc = nlp("Jack and Jill rode up the hill in Les Deux Alpes")
     print([(ent.text, ent.label_) for ent in doc.ents])  # noqa: T201
+
+
+@pytest.mark.skipif(not has_torch_cuda_gpu, reason="needs GPU & CUDA")
```

Comment:
Is Torch/Cuda really necessary for this?

Reply:
Thanks. I also renamed the test.